101-webapp-linux-django/GettingStarted.md

```diff
@@ -0,0 +1,161 @@
+
+And there we go, your deployment is Succeeded. Let's check the resource on the [Azure Portal](https://portal.azure.com).
+
+On the portal, go to Resource Groups. In this blade, you can see the Resource Group that you created.
+
+Click on the Resource Group and there it's the resources
+
+![Screen](./images/portal-resource.png)
+
+Congratulations! You have deployed the template successfully. We can see our web visiting an URL like that:
+
+#### webAppName.azurewebsites.net
+
+### Using the Portal
+
+At the Portal, in All Services look for **Templates**, you can favourite this service.
+
+Click in **Add** to add your template:
+
+On General, type the name and the description for your template, and click on [Ok].
+
+![Screen](./images/aztemplate2.png)
+
+On ARM Template, replace the contents of the template with your template, and click on [OK] and then on [Add].
+
+![Screen](./images/aztemplate3.png)
+
+Click on the refresh button and you will find your template. Click on it and the click in [Deploy].
+
+![Screen](./images/azportaldeploy.png)
+
+On the screen Custom Deployment, inform the values for the parameters, by now you must be already familiar with.
+
+Select [Agree] and click on [Purchase].
+
+![Screen](./images/azportaldeploy2.png)
+
+As you can see, it's deploying.
+
+After a couple of minutes, voilà, you have your Web App deployed.
+
+![Screen](./images/azportaldeploy3.png)
+
+Go to the Resource. Repeat the test that you have done before.
+
+**p.s.: Pretty easy to create resources on Azure, right? But if you are the sort of IT guy that always loves automation, here is the surprise. Just click on the button below and it will automatically deploy the VM through the  Azure Portal.**
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-webapp-linux-django%2Fazuredeploy.json" target="_blank">
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-webapp-linux-django%2Fazuredeploy.json" target="_blank">
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.png"/>
+</a>
+
+#### Important disclaimer: Azure charges you for the resources you are using, and you don't want to finish all your credits at once, right? So, don't forget to stop the ___Service___ at the portal or even delete the Resource Group you have created to avoid unnecessary charges
+
+### Deploy your code
+
+It will be really easy. First of all, download the basic Django project configuration from [Azure-Sample](https://github.com/Azure-Samples/docker-django-webapp-linux.git). You can clone the repository using git.
```

Comment:
This sample doesn't use a production ready Dockerfile, you can reference our VS Code django + docker sample here:
https://code.visualstudio.com/docs/python/tutorial-create-containers#_changes-for-django-apps

Official sample app here:
https://github.com/Microsoft/python-sample-vscode-django-tutorial

Reply:
Thank you for the feedback. I updated the sample app with the official one.